### PR TITLE
Added namespace to all document API calls

### PR DESCRIPTION
**Pull request by sephib**
In order to solve issue #317  this PR adds namespaces to all Document API calls.

If no namespace is supplied it receives the schema value

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: vespa/application.py [hunk 1]
```diff
@@ -397,33 +397,37 @@ def query_batch(
...
+            namespace = schema
+
         with VespaSync(app=self) as sync_app:
             return sync_app.feed_data_point(
-                schema=schema, data_id=data_id, fields=fields
+                namespace=namespace, schema=schema, data_id=data_id, fields=fields
             )
 
-    def _feed_batch_sync(self, schema: str, batch: List[Dict]):
+    def _feed_batch_sync(self, namespace: str, schema: str, batch: List[Dict], **kwargs) -> VespaResponse:
```
Comment (thigm85) [inline vespa/application.py:418]: 1) Let's follow a pattern of adding the `namespace` argument at the end of the parameter list.
2) Why add `**kwargs` here?

File: vespa/application.py [hunk 2]
```diff
@@ -524,31 +538,39 @@ def delete_batch(
...
+            namespace = schema
+
         with VespaSync(self) as sync_app:
             return sync_app.delete_all_docs(
-                content_cluster_name=content_cluster_name, schema=schema
+                content_cluster_name=content_cluster_name, namespace=namespace, schema=schema
             )
 
-    def get_data(self, schema: str, data_id: str) -> VespaResponse:
+    def get_data(self, namespace: str, schema: str, data_id: str) -> VespaResponse:
```
Comment (thigm85) [inline vespa/application.py:573]: The changes on this PR should not break the API. Note that in this case `namespace` is a required argument. It should have a default and use schema when not provided.

You can check API breaks by looking at the failed integration tests triggered by this PR.

**Review by thigm85 — CHANGES_REQUESTED** [review]
I made some remarks. Most importantly, this PR should not break API changes. Please, make sure that there are no integration test failures.

**Comment by sephib** [comment]
Hi @thigm85  -
Should I revert to the original test_integration_docker file?

**Comment by sephib** [comment]
@thigm85 is there anything else that is required to merge the pr?

**Comment by sephib** [comment]
@thigm85 @kkraune  please let me know if there is anything that is required to merge this PR

**Comment by thigm85** [comment]
Hi @sephib, I will take a look at it today

**Comment by thigm85** [comment]
Hi @sephib, I would like to push some changes to your PR. Can you grant me permission to do so?

**Comment by thigm85** [comment]
Hi @sephib, I am still getting permission denied: 

```
remote: Permission to sephib/pyvespa.git denied to thigm85.
```

**Comment by thigm85** [comment]
Since I still lack permission, I created a copy of this PR under pyvespa repo. I will close this one and we can follow the progress here: https://github.com/vespa-engine/pyvespa/pull/337

**Pull request closed without merging**